jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -282,8 +282,13 @@ def test_select_and_gather_add(self, harness: primitive_harness.Harness):
   def test_unary_elementwise(self, harness: primitive_harness.Harness):
     dtype = harness.params["dtype"]
     lax_name = harness.params["lax_name"]
+    if (lax_name in ("acosh", "asinh", "atanh", "bessel_i0e", "bessel_i1e", "digamma",
```

Comment:
Can you please update the jax2tf doc with a note about this?

Reply:
Yes!